### PR TITLE
Fix NGINX ProxyManager (Install / Update)

### LINKED ISSUE #761: Nginx Proxy Manager LXC Error in line 146
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

When running the Nginx Proxy Manager LXC script I get  [ERROR] in line 146: exit code 0: while executing command $STD pnpm run build.  I ran the script twice once default and once advance with only verbose mode on.

### What settings are you currently utilizing?

- [X] Default Settings
- [X] Advanced Settings

### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

``` 
npm notice 
npm notice New major version of npm available! 8.19.4 -> 10.9.2
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.9.2
npm notice Run npm install -g npm@10.9.2 to update!
npm notice 
 ✓ Installed pnpm
Would you like to install an older version (v2.10.4)? <y/N> N
 ✓ Downloaded Nginx Proxy Manager v2.12.1
 ✓ Set up Enviroment

   ╭──────────────────────────────────────────────────────────────────╮
   │                                                                  │
   │                Update available! 8.15.9 → 9.15.0.                │
   │   Changelog: https://github.com/pnpm/pnpm/releases/tag/v9.15.0   │
   │                Run "pnpm add -g pnpm" to update.                 │
   │                                                                  │
   │      Follow @pnpmjs for updates: https://twitter.com/pnpmjs      │
   │                                                                  │
   ╰──────────────────────────────────────────────────────────────────╯

Pr WARN  deprecated messageformat@2.3.0: Package renamed as '@messageformat/core', see messageformat.github.io for more details. 'messageformat@4' will eventually provide a polyfill for Intl.MessageFormat, once it's been defined by Unicode & ECMA.
 WARN  deprecated node-sass@9.0.0: Node Sass is no longer supported. Please use `sass` or `sass-embedded` instead.
PrDownloading github.com/tabler/tabler/00f78ad823311bc3ad974ac3e5b0126198f0a813: 33.90 MB/33.90 MB, doneoading github.com/tabler/tabler/00f78ad823311bc3ad974ac3e5b0126198f0a813: 512.00 B/33.90 MB
PrPackages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: /root/.local/share/pnpm/store/v3
  Virtual store is at:             node_modules/.pnpm
Pr WARN  deprecated popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
Pr WARN  25 deprecated subdependencies found: @npmcli/move-file@1.1.2, @npmcli/move-file@2.0.1, are-we-there-yet@3.0.1, copy-concurrently@1.0.5, core-js@1.2.7, core-js@2.6.12, figgy-pudding@3.5.2, fs-write-stream-atomic@1.0.10, fsevents@1.2.13, gauge@4.0.4, glob@5.0.15, glob@7.1.7, glob@7.2.3, glob@8.1.0, inflight@1.0.6, move-concurrently@1.0.1, npmlog@6.0.2, q@1.5.1, resolve-url@0.2.1, rimraf@2.7.1, rimraf@3.0.2, source-map-resolve@0.5.3, source-map-url@0.4.1, urix@0.1.0, uuid@3.4.0
Packages: +905
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
PrProgress: resolved 909, reused 0, downloaded 907, added 905, done
node_modules/.pnpm/core-js@2.6.12/node_modules/core-js: Running postinstall script, done in 35ms
nonode_modules/.pnpm/ejs@2.7.4/node_modules/ejs: Running postinstall script, done in 29ms
nonode_modules/.pnpm/node-sass@9.0.0/node_modules/node-sass: Running install script, done in 802ms
nonode_modules/.pnpm/node-sass@9.0.0/node_modules/node-sass: Running postinstall script, done in 292ms
 \
devDependencies:
+ @babel/core 7.26.0
+ babel-core 6.26.3
+ babel-loader 8.4.1 (9.2.1 is available)
+ babel-preset-env 1.7.0
+ backbone 1.6.0
+ backbone.marionette 4.1.3
+ copy-webpack-plugin 5.1.2 (12.0.2 is available)
+ css-loader 3.6.0 (7.1.2 is available)
+ ejs-lint 1.2.2 (2.0.1 is available)
+ ejs-loader 0.3.7 (0.5.0 is available)
+ ejs-webpack-loader 2.2.2
+ file-loader 6.2.0
+ html-webpack-plugin 4.5.2 (5.6.3 is available)
+ imports-loader 0.8.0 (5.0.0 is available)
+ jquery 3.7.1
+ jquery-mask-plugin 1.14.16
+ jquery-serializejson 2.9.0 (3.2.1 is available)
+ marionette.approuter 1.0.2
+ marionette.templatecache 1.0.0
+ messageformat 2.3.0 deprecated
+ messageformat-loader 0.8.1
+ mini-css-extract-plugin 0.9.0 (2.9.2 is available)
+ moment 2.30.1
+ node-sass 9.0.0 deprecated
+ nodemon 2.0.22 (3.1.7 is available)
+ numeral 2.0.6
+ sass-loader 10.5.2 (16.0.4 is available)
+ style-loader 1.3.0 (4.0.0 is available)
+ tabler-ui 0.0.31
+ underscore 1.13.7
+ webpack 4.47.0 (5.97.1 is available)
+ webpack-cli 3.3.12 (5.1.4 is available)
+ webpack-visualizer-plugin 0.1.11

 WARN  Issues with peer dependencies found
.
└─┬ marionette.templatecache 1.0.0
  ├── ✕ unmet peer backbone@~1.3.3: found 1.6.0
  ├── ✕ unmet peer underscore@~1.8.3: found 1.13.7
  └─┬ backbone.marionette 4.0.0-beta.1
    ├── ✕ unmet peer backbone@~1.3.3: found 1.6.0
    └── ✕ unmet peer underscore@">= 1.8.3 <= 1.9.x": found 1.13.7

Done in 9.1s
 | WARN  deprecated messageformat@2.3.0: Package renamed as '@messageformat/core', see messageformat.github.io for more details. 'messageformat@4' will eventually provide a polyfill for Intl.MessageFormat, once it's been defined by Unicode & ECMA.
 WARN  deprecated node-sass@9.0.0: Node Sass is no longer supported. Please use `sass` or `sass-embedded` instead.
Pr WARN  deprecated popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
Pr WARN  25 deprecated subdependencies found: @npmcli/move-file@1.1.2, @npmcli/move-file@2.0.1, are-we-there-yet@3.0.1, copy-concurrently@1.0.5, core-js@1.2.7, core-js@2.6.12, figgy-pudding@3.5.2, fs-write-stream-atomic@1.0.10, fsevents@1.2.13, gauge@4.0.4, glob@5.0.15, glob@7.1.7, glob@7.2.3, glob@8.1.0, inflight@1.0.6, move-concurrently@1.0.1, npmlog@6.0.2, q@1.5.1, resolve-url@0.2.1, rimraf@2.7.1, rimraf@3.0.2, source-map-resolve@0.5.3, source-map-url@0.4.1, urix@0.1.0, uuid@3.4.0
Already up to date
Progress: resolved 909, reused 907, downloaded 0, added 0, done
 | WARN  Issues with peer dependencies found
.
└─┬ marionette.templatecache 1.0.0
  ├── ✕ unmet peer backbone@~1.3.3: found 1.6.0
  ├── ✕ unmet peer underscore@~1.8.3: found 1.13.7
  └─┬ backbone.marionette 4.0.0-beta.1
    ├── ✕ unmet peer backbone@~1.3.3: found 1.6.0
    └── ✕ unmet peer underscore@">= 1.8.3 <= 1.9.x": found 1.13.7

Done in 3.3s
 \
> nginx-proxy-manager@2.12.1 build /nginx-proxy-manager-2.12.1/frontend
> webpack --mode production

 |(node:7495) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
(Use `node --trace-deprecation ...` to show where the warning was created)
 \Hash: 1f6cb777e2244c6bc2be
Version: webpack 4.47.0
Time: 322ms
Built at: 12/10/2024 2:33:06 AM
 2 assets
Entrypoint main = js/main.bundle.js?v=2.12.1
Entrypoint login = js/login.bundle.js?v=2.12.1
[0] ./js/index.js 5.47 KiB {1} [built] [failed] [1 error]
[1] ./js/login.js 5.47 KiB {0} [built] [failed] [1 error]

ERROR in ./js/index.js
Module build failed (from ./node_modules/.pnpm/babel-loader@8.4.1_@babel+core@7.26.0_webpack@4.47.0/node_modules/babel-loader/lib/index.js):
Error: [BABEL] /nginx-proxy-manager-2.12.1/frontend/js/index.js: Cannot find module '/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+generator@7.26.3/node_modules/@jridgewell/gen-mapping/dist/gen-mapping.umd.js'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:1055:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:1048:15)
    at resolveExports (node:internal/modules/cjs/loader:556:14)
    at Function.Module._findPath (node:internal/modules/cjs/loader:596:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1014:27)
    at Function.Module._load (node:internal/modules/cjs/loader:873:27)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+generator@7.26.3/node_modules/@babel/generator/lib/source-map.js:7:19)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+generator@7.26.3/node_modules/@babel/generator/lib/index.js:7:18)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+traverse@7.26.4/node_modules/@babel/traverse/lib/path/index.js:14:18)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+traverse@7.26.4/node_modules/@babel/traverse/lib/context.js:7:14)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+traverse@7.26.4/node_modules/@babel/traverse/lib/traverse-node.js:7:16)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+traverse@7.26.4/node_modules/@babel/traverse/lib/path/context.js:27:21)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)

ERROR in ./js/login.js
Module build failed (from ./node_modules/.pnpm/babel-loader@8.4.1_@babel+core@7.26.0_webpack@4.47.0/node_modules/babel-loader/lib/index.js):
Error: [BABEL] /nginx-proxy-manager-2.12.1/frontend/js/login.js: Cannot find module '/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+generator@7.26.3/node_modules/@jridgewell/gen-mapping/dist/gen-mapping.umd.js'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:1055:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:1048:15)
    at resolveExports (node:internal/modules/cjs/loader:556:14)
    at Function.Module._findPath (node:internal/modules/cjs/loader:596:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1014:27)
    at Function.Module._load (node:internal/modules/cjs/loader:873:27)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+generator@7.26.3/node_modules/@babel/generator/lib/source-map.js:7:19)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+generator@7.26.3/node_modules/@babel/generator/lib/index.js:7:18)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+traverse@7.26.4/node_modules/@babel/traverse/lib/path/index.js:14:18)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+traverse@7.26.4/node_modules/@babel/traverse/lib/context.js:7:14)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+traverse@7.26.4/node_modules/@babel/traverse/lib/traverse-node.js:7:16)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/@babel+traverse@7.26.4/node_modules/@babel/traverse/lib/path/context.js:27:21)
    at Module._compile (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (/nginx-proxy-manager-2.12.1/frontend/node_modules/.pnpm/v8-compile-cache@2.4.0/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
Child HtmlWebpackCompiler:
     2 assets
    Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
    Entrypoint HtmlWebpackPlugin_1 = __child-HtmlWebpackPlugin_1
    [0] ./node_modules/.pnpm/ejs-webpack-loader@2.2.2/node_modules/ejs-webpack-loader!./html/index.ejs 6.52 KiB {0} [built]
    [1] ./node_modules/.pnpm/ejs-webpack-loader@2.2.2/node_modules/ejs-webpack-loader!./html/login.ejs 6.69 KiB {1} [built]
 ELIFECYCLE  Command failed with exit code 2.

[ERROR] in line 146: exit code 0: while executing command $STD pnpm run build 
```

### Please provide detailed steps to reproduce the issue.

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/nginxproxymanager.sh)"

### PR DESCRIPTION
**Pull request by MickLesk**
## Description
Reverted deprecated pnpm in frontend, switched to official yarn

Fixes #761 

## Type of change
Please check the relevant option(s):

- [x] Bug fix (non-breaking change that resolves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would cause existing functionality to change unexpectedly)
- [ ] New script (a fully functional and thoroughly tested script or set of scripts.)

## Prerequisites
The following efforts must be made for the PR to be considered. Please check when completed:
- [x] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [x] Testing performed (I have tested my changes, ensuring everything works as expected)
- [ ] Documentation updated (I have updated any relevant documentation)

## Additional Information (optional)
Provide any additional context or screenshots about the feature or fix here.

## Related Pull Requests / Discussions

If there are other pull requests or discussions related to this change, please link them here:
- Related PR #

### CODE REVIEW

**Comment by havardthom** [comment]
What is the bug? Script installs fine here.
image

File: install/nginxproxymanager-install.sh [hunk 1]
```diff
@@ -141,9 +141,9 @@ msg_ok "Set up Enviroment"
 
 msg_info "Building Frontend"
 cd ./frontend
-$STD pnpm install
-$STD pnpm upgrade
-$STD pnpm run build
+$STD yarn cache clean --silent --force
```
Comment (havardthom) [inline install/nginxproxymanager-install.sh:144]: Have you tested this? Where is yarn installed? pnpm install section is not removed? https://github.com/tteck/Proxmox/commit/edf3803362cb7ad152fb386997def01edf5ef57e#diff-e676f31f25717f8123cc0904ec5973dee3dc53b86e2b0ff7bec0eeb72c6ce82fR132

Comment (MickLesk) [reply install/nginxproxymanager-install.sh:144]: Yep, you can run the bash script against the develop Branch

**Comment by MickLesk** [comment]
> What is the bug? Script installs fine here.
> image
> 

Install works, Update not. Because pnpm load deprecated Files, for first Install is this Not a big Problem, but with Update

File: install/nginxproxymanager-install.sh (hunk 1 above)
Comment (MickLesk) [reply install/nginxproxymanager-install.sh:144]: https://github.com/NginxProxyManager/nginx-proxy-manager/blob/b3de76c945b087b9a585faff128ca655170f8ea1/docker/Dockerfile#L44-L47

Comment (rowie) [reply install/nginxproxymanager-install.sh:144]: ✓ Stopped Services
 ✓ Cleaned Old Files
 ✓ Downloaded NPM v2.12.1
 ✓ Setup Enviroment
 / Building Frontend   
[ERROR] in line 143: exit code 0: while executing command yarn cache clean --silent --force &> /dev/null

running against the dev branch ...

Comment (MickLesk) [reply install/nginxproxymanager-install.sh:144]: Lol. Work on my systems fine :D fck NPM

**Comment by rowie** [comment]
Why is this thing dying with every update?

**Comment by havardthom** [comment]
Update works fine here also, service runs fine afterwards.
The issue linked in the description had an error in install script which I can not reproduce: `[ERROR] in line 146: exit code 0: while executing command $STD pnpm run build`
Does anyone else have issues when installing/updating current script?
image

File: install/nginxproxymanager-install.sh (hunk 1 above)
Comment (havardthom) [reply install/nginxproxymanager-install.sh:144]: > https://github.com/NginxProxyManager/nginx-proxy-manager/blob/b3de76c945b087b9a585faff128ca655170f8ea1/docker/Dockerfile#L44-L47

That is backend install, not the same as frontend install which you are changing in this PR

**Comment by towerhand** [comment]
No problems on my end with the original script @havardthom.

**Comment by tuxmontero** [comment]
Coming to help here.
The update was failing with the same message for me:
[ERROR] in line 137: exit code 0: while executing command python3 -m pip install --no-cache-dir certbot-dns-cloudflare &> /dev/null

I'm not a python expert but searching I fixed it with this:
# creating virtual env
python3 -m venv myenv

# Activate the venv
source myenv/bin/activate

# upgrading pip
pip install --upgrade pip

Then the update command would work

**Pull request closed without merging**